hapi/src/api/account.api.js

```diff
@@ -272,6 +304,29 @@ const grantConsent = async account => {
   return consentTransaction
 }
 
+const formatSchedule = (schedule) => {
+  let scheduleFormat = ''
+
+  let hours
+  for (hours of schedule)
+    scheduleFormat += `, ${hours.day} ${hours.open} - ${hours.close}`
+
+  return scheduleFormat.replace(',', ' ')
+}
+
+const formatLifebankData = (lifebankData) => {
+  lifebankData.schedule = formatSchedule(JSON.parse(lifebankData.schedule))
+  lifebankData.coordinates = JSON.parse(lifebankData.coordinates)
+  if (lifebankData.immunity_test) lifebankData.immunity_test = 'Yes'
```

Comment:
What if we move this to lifebank.api.js file?

Reply:
ok good